.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "pwa-chrome",
```

Comment:
why is this file changed ?

Reply:
I think this file was missing and I tried to run vscode debugger a few days ago, and it created this file, I can either delete it or add it to gitignore